ethereum/spec/src/property-test/java/tech/pegasys/teku/spec/logic/versions/deneb/helpers/CKZG4844PropertyTest.java

```diff
@@ -31,36 +31,34 @@
 
 @AddLifecycleHook(KzgResolver.class)
 public class CKZG4844PropertyTest {
-
   @Property(tries = 100)
-  void fuzzComputeAggregateKzgProof(
+  void fuzzVerifyAggregateKzgProof(
```

Comment:
guess rename to fuzzVerifyBlobKzgProofBatch

Reply:
Copy-paste will kill us one day